llvm/test/DebugInfo/X86/dynamic-bitfield.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: llc -mtriple=x86_64 -O0 -filetype=obj -o - %s | llvm-dwarfdump -v -debug-info - | FileCheck %s
+; RUN: llvm-as < %s | llvm-dis | llc -mtriple=x86_64 -O0 -filetype=obj -o - | llvm-dwarfdump -v -debug-info - | FileCheck %s
```

Comment:
Maybe have the as/dis roundtrip as a separate RUN line with a comment explaining why it's there?

Reply:
(I assume this crashes without the patch? And otherwise this doesn't change the IR printing? I'm of two minds about whether it might be good to have an actual textual IR dump with CHECK lines - there's something we're printing with this patch that we weren't printing without it that might be worth more directly validating, rather than only indirectly via parsing/doing some other testing)

Reply:
Makes sense to be more explicit IMO.

Reply:
Ok, I looked at this again, and I think perhaps just a comment before the `RUN` line would be enough.  The reason is that adding the `llvm-dis` is exercising round-tripping of the IR, and there's a `CHECK` later in the file that verifies that the correct value (an expression as opposed to an integer) comes through.